articles/extensions/authorization-extension/v2/index.md

```diff
@@ -1,358 +1,64 @@
 ---
 toc: true
-description: This page explains how to setup and manage the Authorization Extension v2.
+title: Authorization Extension
+description: Control user authorization behavior during runtime with the Authorization Extension
 ---
-# Auth0 Authorization Extension v2
 
-The Auth0 Authorization Extension provides user authorization support in Auth0. Currently the extension supports authorizations for Users using Groups, Roles and Permissions.
+<div class="topic-page-header">
+  <div data-name="example" class="topic-page-badge"></div>
+  <h1>Authorization Extension</h1>
+  <p>
+    The Authorization Extension provides support for user authorization via Groups, Roles, and Permissions. You can define the expected behavior during the login process, and your configuration settings will be captured in a [rule](/rules) that's executed during runtime.
```

Comment:
The page looks "funny" https://auth0-docs-content-pr-5025.herokuapp.com/docs/extensions/authorization-extension/v2

![image](https://user-images.githubusercontent.com/11715799/31224902-69046210-a9d8-11e7-8107-ccc00cc2a953.png)

Reply:
FYI: This is what happens when you don't include `classes: topic-page` in the metadata area. :(